src/js/components/icons/Pulse.js

```diff
@@ -0,0 +1,32 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+import Add from './base/Add';
+import CSSClassnames from '../../utils/CSSClassnames';
+import classnames from 'classnames';
+
+const CLASS_ROOT = CSSClassnames.PULSE;
+
+export default class Pulse extends Component {
+  render () {
+    const classes = (this.props.className)
+      ? classnames(
+          CLASS_ROOT,
+          this.props.className
+        )
+      : CLASS_ROOT;
+
+    return (
+      <div className={classes}>
+        <div className={`${CLASS_ROOT}__icon`}>
+          <Add />
```

Comment:
@marlonpp gave a great idea, is that possible to make the icon dynamic? We can definitely default to Add, but I wonder if there are cases that I can use the pulse icon with other controls.